models/api.model.js

```diff
@@ -17,3 +17,13 @@ exports.findArticleById = (articleId) => {
 			}
 		});
 };
+
+exports.fetchArticles = () => {
+	return db
+		.query(
+			`SELECT articles.article_id, articles.title, articles.author, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT OUTER JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC;`
+		)
```

Comment:
I'd take a look at the [CAST](https://www.w3schools.com/sql/func_sqlserver_cast.asp) function for the comment_count